apps/zui/src/electron/start.test.ts

```diff
@@ -14,14 +14,6 @@ jest.mock("@brimdata/zed-node")
 
 afterEach(teardown)
 
-test("start is called in zed lake", async () => {
-  const appMain = (await main({
-    devtools: false,
-    autoUpdater: false,
-  })) as MainObject
-  expect(appMain.lake.start).toHaveBeenCalledTimes(1)
-})
-
```

Comment:
This test was causing problems and is not necessary. If the lake doesn't start, many many other tests will fail.